### DISCUSSION TITLE
怎样设置飞书消息通知到工单的发起人，定时任务怎样设置

### DISCUSSION THREAD

**Discussion by planet-zjx** [hhyo/Archery#1685] [Q&A]
1、目前通过配置 资源组添加飞书webhook地址 来实现工单上线和审批消息通知到飞书群，怎样设置把工单整个流程中的消息通知到工单的发起人，审批人呢？
2、定时任务用DJANGO Q实现，由于不熟练，怎样设置，一个工单超过1天没有审批，自动发消息通知对应审批节点的负责人呢？

以上两个问题，麻烦大佬给解答下，十分感谢。

**Comment by hhyo** [comment]
1. 暂不支持，可自行开发

2.参考django_q文档，很详细的使用教程 https://django-q.readthedocs.io/en/latest/index.html

**Reply by planet-zjx** [reply]
好的，多谢
